packages/create-single-spa/package.json

```diff
@@ -8,7 +8,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "generator-single-spa": "^1.17.1",
+    "generator-single-spa": "1.17.1",
```

Comment:
Why was this change made?

Reply:
This locks the version of generator-single-spa in create-single-spa so that they're always in lockstep. Otherwise, using an older version (eg. npx create-single-spa@1.14.1) will still use the latest minor version of generator-single-spa (eg. 1.17.0).